packages/protvista-datatable/src/protvista-datatable.ts

```diff
@@ -1,26 +1,91 @@
-import { LitElement, html } from "lit-element";
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  CSSResult,
+  PropertyDeclarations,
+} from "lit-element";
 import { v1 } from "uuid";
 import { ScrollFilter } from "protvista-utils";
-/* eslint-disable import/extensions, import/no-extraneous-dependencies */
+// Not sure why, but eslint doesn't see it...
```

Comment:
With the following in `.eslintrc.json`:
```
  "settings": {
    "import/resolver": {
      "node": {
        "extensions": [".js", ".ts",".d.ts"]  // <---with ".d.ts" now
      }
    }
  }
```
the linter shouldn't complain anymore.

Reply:
You, sir, are a gentleman and a scholar. I'd assumed ".ts" would have covered ".d.ts", but no.